srcpkgs/vscodium-bin/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'vscodium-bin'
+pkgname=vscodium-bin
+version=1.33.0
+revision=1
+archs="i686 x86_64"
+depends="GConf libXtst libxkbfile nss dejavu-fonts-ttf"
+short_desc="Binary releases of VS Code without MS branding/telemetry/licensing"
+maintainer="shizonic <realtiaz@gmail.com>"
+license="MIT"
+homepage="https://github.com/VSCodium/vscodium"
+nostrip_files="vscodium"
+
+if [ "${XBPS_TARGET_MACHINE}" = "x86_64" ]; then
+	_archive="VSCodium-linux-x64-${version}.tar.gz"
+	checksum=388506a0bee87a72102bf20d32efaf18d6f5d29286bdda3684f63d87e63c646f
+else
+	_archive="VSCodium-linux-ia32-${version}.tar.gz"
+	checksum=cd2e8dddbb5858bc796279f2727c4ee6794653ddd0b6b4316d05334d4c7f2648
+fi
+distfiles="https://github.com/VSCodium/vscodium/releases/download/${version}/${_archive}"
+
+do_extract() {
+	local src="$XBPS_SRCDISTDIR/$pkgname-$version/$_archive"
+	mkdir -p vscodium
```

Comment:
The directory where you extract to has to be the same as `wrksrc` otherwise `xbps-src` won't clean up after the build is finished.

Reply:
Ok. Thx. Gonna fix this.